src/utils/index.js

```diff
@@ -1,12 +1,13 @@
-// This file contains utility functions that are exposed game-side
+/** This file contains utility functions that are exposed game-side */
 
 /**
   * Choose a random number between a and b.
   * @example randRange([a, b], bipolar)
   * @example randRange(a, b, bipolar)
-  * @arg {number} a
-  * @arg {number} b
-  * @arg {boolean} [bipolar] - If true, then the returned value will be inverted half of the time
+  * @global
+  * @arg     {number}  a
+  * @arg     {number}  b
+  * @arg     {boolean} [bipolar=false] Should the returned value will be inverted half of the time
   * @returns {number}
   */
 randRange = function(a, b, bipolar) {
```

Comment:
Whether the functionality is useful can be debated, but bipolar DOES do different things than what you suggested. Bipolar 10, 20 would give a range from either 10 to 20, or -20 to -10. If you use randomRange, then that would include the values -10 to 10, which bipolar would not.